automatic/dropbox/update_helper.ps1

```diff
@@ -1,36 +1,30 @@
 
 function drpbx-compare {
-	param(
-		[string]$nonstable_version
-	)
+	param( 
+[Parameter(Position = 0)][string]$_version, [string]$build = 'stable' )
     $releases = 'https://www.dropboxforum.com/t5/Desktop-client-builds/bd-p/101003016'
-    $HTML = Invoke-WebRequest -UseBasicParsing -Uri $releases
-    $stable_builds = @()
-    $HTML.Links | foreach {
-  	  if ($_.href -match "stable" ) { $stable_builds += $_.href }
-    }
-    $re_dash = '-'; $re_dashndigits = "\-\D+"; $re_t5 = 't5'; $re_abc = '[a-z]\w+'; $re_num_M = '\d+\#[M]\d+';
-    $re_dot = '.'; $re_non = ''; $re_stable = 'Stable-'; $re_build = 'Build-';
-    $stable_version = ( drpbx-builds -hrefs $stable_builds -nonstable_version $nonstable_version )
-    return $stable_version
+    $weblinks = "${env:temp}\${build}links.txt"
+    $HTML = (Invoke-WebRequest -UseBasicParsing -Uri $releases).Links`
+     | where {($_ -match $build)} | Select -First 6 | Out-File $weblinks
```

Comment:
why writing this out to a separate file?
Why not just set a variable containing the links instead (basically just removing `| Out-File $weblinks`)?